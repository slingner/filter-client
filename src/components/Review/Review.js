import React from 'react';

const Review = (props) => {
  return (
    <li>
      {props.reviewText}
      <button className='delete-review-button' onClick={() => props.deleteReview(props.reviewId)}>Delete</button>
    </li>
  )
}

export default Review;