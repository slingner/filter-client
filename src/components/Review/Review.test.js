import React from 'react'
import ReactDOM from 'react-dom'
import Review from './Review'
import renderer from 'react-test-renderer';


describe(`<Login />`, () => {
  //Smoke Testing
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <Review 
        
      />,
      div
    );
    ReactDOM.unmountComponentAtNode(div)
  })
  //Snapshot Testing
  it('renders UI as expected', () => {
    const tree = renderer
      .create(<Review />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})