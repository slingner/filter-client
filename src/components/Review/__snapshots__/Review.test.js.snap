// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> renders UI as expected 1`] = `
<li>
  <button
    className="delete-review-button"
    onClick={[Function]}
  >
    Delete
  </button>
</li>
`;
